packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -28,13 +28,13 @@ import type {CategorizerPromptJSON} from "../../widget-ai-utils/categorizer/cate
 import type {PerseusCategorizerWidgetOptions} from "@khanacademy/perseus-core";
 
 type Props = WidgetProps<RenderProps, PerseusCategorizerScoringData> & {
-    values: ReadonlyArray<string>;
+    values: ReadonlyArray<number>;
```

Comment:
This should have been `ReadonlyArray<number>` all along. Fixing this let me remove the `@ts-expect-error` below.

Reply:
I thought this might be the case, but I couldn't figure out why it was added at all. It's already included in renderProps, so I thought it might have been for another reason. Glad we can make it number! Do we even need that added on if it's already added via RenderProps?